lib/Dialect/TritonGPU/Transforms/Pipeliner/MatmulLoopPipeline.cpp

```diff
@@ -220,9 +220,11 @@ getBlockedEncoding(tt::LoadOp loadOp, ModuleAxisInfoAnalysis &axisInfo) {
   AxisInfo::DimVectorT contiguity = axisInfo.getAxisInfo(src)->getContiguity();
   SmallVector<unsigned> order = argSort(contiguity);
   unsigned currPerThread = getNumElementsPerThread(loadOp, order, axisInfo);
+  SmallVector<unsigned, 4> sizePerThread(order.size(), 1);
```

Comment:
Nit, any reason we explicitly make the SmallVector size 4 when above `order` has the default size?  Seems like one of these should be right.

Reply:
The only reason is that it was copied from existing code that also calls `getNumElementsPerThread`. I changed both code to  `SmallVector<unsigned>`